src/plugins/expressions/public/services.ts

```diff
@@ -18,22 +18,15 @@
  */
 
 import { NotificationsStart } from 'kibana/public';
-import { createKibanaUtilsCore, createGetterSetter } from '../../kibana_utils/public';
```

Comment:
I noticed this is the only remaining usage of `createKibanaUtilsCore` in the repo, so I've removed that in this PR as well.

Considering that it isn't used elsewhere and is also based on `createGetterSetter` which we've been moving away from, I thought it was safe to remove... but please let me know if there's any reason you feel we need to keep it.